test-case-creation-strategy.mdx

```diff
@@ -0,0 +1,189 @@
+     <img
+       src="images/dependency-01-accept-cookies.png"
+       alt="no dependency for accept cookies"
+       width="300px"
+     />
+   </Frame>
+
+2. The `login with username and password` test case is based on the `accept cookies` test case.
+
+   <Frame caption="login with username and password is based on the accept cookies, screenshot 02/2024">
+     <img
+       src="images/dependency-02-login.png"
+       alt="login with username and password is based on the accept cookies"
+       width="300px"
+     />
+   </Frame>
+
+3. The `open profile` test case is based on the `login with username and password` test case.
+   <Frame caption="Open profile is based on login with username and password, screenshot 02/2024">
+     <img
+       src="images/dependency-03-open-profile.png"
+       alt="Open profile is based on login with username and password"
+       width="300px"
+     />
+   </Frame>
+
+<Tip>
+**Advantages of using dependencies**
+
+- code is reused and only maintained in one place
+- much shorter test cases which are easier to understand and debug
+- much easier for the AI agent to be successful during code generation since it can concentrate on the main task instead of all the stuff before
+
+</Tip>
+
+## Help the agent out if it went wrong
+
+We are constantly improving the agent but it still might go wrong. In this case you need to help it find its way.
+Use the step inspector to fine tune interactions and assertions.
+
+### Add missing steps
+
+You can add missing steps by clicking the `+` icon and choosing the desired intearction or assertion.
+
+<Frame caption="add missing steps, screenshot 02/2024">
+  <img src="images/steps-add-step.png" alt="add missing steps" width="300px" />
+</Frame>
+
+### Delete unnecessary steps
+
+You can easily delete unnecessary steps by clicking the trash can symbol.
+
+<Frame caption="delete obsolet steps, screenshot 02/2024">
+  <img
+    src="images/steps-delete-step.png"
+    alt="delete obsolet steps"
+    width="300px"
+  />
+</Frame>
+
+### Run test cases
+
+Running a test case is producing a Trace. A Trace is a detailed description of what happened during test case execution. Use the Trace for debugging.
+
+<Frame caption="run test case for validation, screenshot 02/2024">
+  <img
+    src="images/run-test-case.png"
+    alt="run test case for validation"
+    width="300px"
+  />
+</Frame>
+
+<Tip>
+  **Advantages of helping out the agent**
+  
+  - modifying steps manually is tpically much faster then changing the prompts
+  - You have much more control over the outcome.
+
+</Tip>
+
+## Use `run locally`
+
+`run locally` helps you to save execution time and you can directly see what is happening.
+You can find the `run locally` in the debug section of your test case.
+
+<Frame caption="run locally: copy command view, screenshot 02/2024">
+  <img
+    src="images/run-locally-command-mobile.png"
+    alt="run locally: copy command view"
```

Comment:
the image btw is NOT the copy command view

Reply:
ok, I will update it on the next run.